exp/lighthorizon/index/cmd/single/main.go

```diff
@@ -187,142 +142,62 @@ func main() {
 	if err := wg.Wait(); err != nil {
 		panic(err)
 	}
-	log.Infof("Uploading indexes")
+
+	postProgress("Reading ledgers",
+		uint64(ledgerCount), uint64(ledgerCount), startTime)
+
+	// Assertion for testing
+	if processed != uint64(ledgerCount) {
+		log.Fatalf("processed %d but expected %d", processed, ledgerCount)
+	}
+
+	log.Infof("Processed %d ledgers via %d workers", processed, parallel)
+	log.Infof("Uploading indices to %s", *targetUrl)
 	if err := indexStore.Flush(); err != nil {
 		panic(err)
 	}
 }
 
-func participantsForOperations(transaction ingest.LedgerTransaction, onlyPayments bool) ([]string, error) {
-	var participants []string
+func postProgress(prefix string, done, total uint64, startTime time.Time) {
```

Comment:
```suggestion
func printProgress(prefix string, done, total uint64, startTime time.Time) {
```
`printProgress` maybe? `post` says, "http request" to me.

Reply:
yeah, good call